CMakeLists.txt

```diff
@@ -202,25 +213,46 @@ if("${DDC_BUILD_KERNELS_SPLINES}")
         find_package(KokkosKernels REQUIRED)
     endif()
 
-    target_link_libraries(DDC INTERFACE Kokkos::kokkoskernels)
-    target_compile_definitions(DDC INTERFACE "DDC_BUILD_KERNELS_SPLINES")
+    add_library(ddc_splines INTERFACE)
+    add_library(DDC::splines ALIAS ddc_splines)
+    install(FILES cmake/FindLAPACKE.cmake DESTINATION lib/cmake/DDC)
```

Comment:
It would be cleaner to use GNUInstallDirs

Reply:
Yes good idea I will prepare a new PR